idaes/generic_models/properties/swco2.py

```diff
@@ -258,14 +257,13 @@ def rule_tc(b, p):
             )
             Ts = T/(251.196*pyunits.K)
             G = sum(b[i]/Ts**i for i in b)
-            return pyunits.convert(
-                (0.475598*pyunits.mW/pyunits.K**(3/2)/pyunits.m *
-                 sqrt(T)*(1 + 2.0/5.0*cint_over_k)/G) +
-                params.tc_d_1*rho[p] +
-                params.tc_d_2*rho[p]**2 +
-                params.tc_d_3*rho[p]**3 +
-                params.tc_d_4*rho[p]**4,
-                to_units=pyunits.W/pyunits.K/pyunits.m)
+            return (
+                0.475598/1e3 * pyunits.W/pyunits.K/pyunits.m *
+                   sqrt(T/pyunits.K)*(1 + 2.0/5.0*cint_over_k)/G +
```

Comment:
Why is divided by `pyunits.K` ? Should it be multiplied?

Reply:
I assume you mean the T in the square root.  The /K is right.  T has units, I'm taking them off.  It that or divide by another sqrt(K) where the other units are.